guardian/configuration-magic
# Provide builds for play 2.7 and 2.8

## What does this change?

Adds builds for play 2.7 and 2.8, fixing a couple of deprecation warnings to allow compilation.

## How to test

Check `sbt clean +package`

The tests in the repo no longer run for me, I receive the error `Error: Unable to access jarfile <HOMEDIR>/configuration-magic/dynamodb-local/DynamoDBLocal.jar` and then an infinite loop `Waiting for dyanmodb[sic] local to boot on port 8000`. I'm not sure of a good alternative to get the tests running again.
